content/articles/javascript-math-anim/index.md

```diff
@@ -30,24 +30,24 @@ We will use these two functions to rotate an image in our animation. Find more a
 This is the JavaScript code.
 
 ```javascript
-    // The image variable
-    let image = document.querySelector("img");
-    // 90 degrees
-    let angle = Math.PI / 2;
+// The image variable
+let image = document.querySelector("img");
+// 90 degrees
+let angle = Math.PI / 2;
 
-    function animate(currentTimeParam, prevTimeArg) {
+function animate(currentTimeParam, prevTimeArg) {
 
-        if (prevTimeArg != null) {
-            angle += (currentTimeParam - prevTimeArg) * 0.004;
-        }
-        // Setting the previous time to the time the function currently fires
-        prevTimeArg = currentTimeParam ;
-        image.style.top = (Math.sin(angle) * 150) + "px";
-        image.style.left = (Math.cos(angle) * 150) + "px";
-        // Passing the current time and previous times
-        requestAnimationFrame(currentTime => animate(currentTime, prevTimeArg));
+    if (prevTimeArg != null) {
+        angle += (currentTimeParam - prevTimeArg) * 0.004;
     }
-    requestAnimationFrame(animate);
+    // Setting the previous time to the time the function currently fires
+    prevTimeArg = currentTimeParam ;
+    image.style.top = (Math.sin(angle) * 150) + "px";
+    image.style.left = (Math.cos(angle) * 150) + "px";
+    // Passing the current time and previous times
+    requestAnimationFrame(currentTime => animate(currentTime, prevTimeArg));
+}
+requestAnimationFrame(animate);
```

Comment:
It's a little unclear what is being passed in this first call to the `animate` function. I would leave a comment explaining it. I shouldn't have to assume what is being passed in the first call to this function.